runtime/compiler/codegen/CodeGenGPU.cpp

```diff
@@ -93,19 +93,33 @@ static const char * nvvmOpCodeNames[] =
    "store",          // TR::fstore
    "store",          // TR::dstore
    "store",          // TR::astore
-
-   NULL,          // TR::wrtbar
-
    "store",          // TR::bstore
    "store",          // TR::sstore
+
+   "wrtbar",         // TR::iwrtbar
```

Comment:
why are these not printing their proper names? making them all print the same seems a bad idea

Reply:
Thanks for pointing this out. changed to NULL according to Gita's comments. https://github.com/eclipse/openj9/pull/3138#issuecomment-427132679